wasmtime-jni/src/io_github_kawamuray_wasmtime_Linker/imp.rs

```diff
@@ -66,6 +67,48 @@ impl<'a> JniLinker<'a> for JniLinkerImpl {
         Ok(ret)
     }
 
+    fn native_externs(
+        env: &JNIEnv,
+        this: JObject,
+        store_ptr: jlong,
+        module: JString,
+    ) -> Result<jobjectArray, Self::Error> {
+        let mut store = interop::ref_from_raw::<Store<StoreData>>(store_ptr)?;
+        let linker = interop::get_inner::<Linker<StoreData>>(env, this)?;
+        let module = utils::get_string(env, *module)?;
+        let mut vec: Vec<String> = Vec::new();
+        for item in linker.iter(&mut *store) {
+            if item.0.eq(&module) {
+                vec.push(item.1.to_string())
+            }
+        }
+        let ret = env.new_object_array(vec.len() as i32, "java/lang/Object", JObject::null())?;
+        for (i, item) in vec.iter().enumerate() {
+            let value: JString = env.new_string(item).unwrap().into();
```

Comment:
You can use `?` operator instead of `unwrap()` as the error conversion from JNI error to wasmtime-java error is defined.

Reply:
OK. I had no idea.